content/sites/reference.md

```diff
@@ -29,10 +29,12 @@ Worker Sites require the latest version of [Wrangler](https://github.com/cloudfl
 
 There are a few specific configuration settings for Workers Sites in your `wrangler.toml`:
 
-| Key           | Value                                                                              | Example                          |
-| ------------- | ---------------------------------------------------------------------------------- | -------------------------------- |
-| `bucket`      | The directory containing your static assets, path relative to your `wrangler.toml` | `bucket = "./public"`              |
-| `entry-point` | The location of your Worker script, default is `workers-site`                      | `entry-point = "./workers-site"` |
+| Key           | Value                                                                              | Example                          | Required |
+| ------------- | ---------------------------------------------------------------------------------- | -------------------------------- | -------- |
+| `bucket`      | The directory containing your static assets, path relative to your `wrangler.toml` | `bucket = "./public"`            | yes      |
+| `entry-point` | The location of your Worker script, default is `workers-site`                      | `entry-point = "./workers-site"` | yes      |
+| `include`     | A list of gitignore-style patterns for files or directories in `bucket` you exclusively want to upload. | `include = ["upload_dir"]` | no |
```

Comment:
maybe we should add a link to the include/exclude section of the docs here for more info